docs/source/_static/install_depthai.sh

```diff
@@ -142,20 +139,13 @@ if [[ $(uname -s) == "Darwin" ]]; then
   git checkout main
   git pull
 
-  # install python 3.10 and python dependencies
-  brew update
 
   if [ "$install_python" == "true" ]; then
     echo "installing python 3.10"
     brew install python@3.10
     python_executable=$(which python3.10)
   fi
 
-  # pip does not have pyqt5 for arm
-  if [[ $(uname -m) == 'arm64' ]]; then
-    echo "Installing pyqt5 with homebrew."
-    brew install pyqt@5
-  fi
```

Comment:
This is not true anymore, hence why it was removed?

Reply:
the pyqt5 was not available for m1 macs on pip, only on brew. This is no longer true, standard
```pip install PyQt5```works now